documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Everything from "This is a high performance" seems unrelated to TF-DF (explaining GPUs, referencing NNs, author anecdote, etc).</p><p> </p><p>What's your goal? Are you looking to introduce TF Datasets? If so, in this guide it's only used in this step here, so the explanation should be similarly short and focused.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>